src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const peopleList = document.querySelector('ul');
+
+function sortBySallary(list) {
+  const items = Array.from(list.children);
+
+  items.sort((a, b) => {
+    const salaryA = parseFloat(
+      a.getAttribute('data-salary').replace(/[\\$,]/g, ''),
```

Comment:
The regular expression in `replace(/[\\$,]/g, '')` is incorrect. Double backslashes are not needed here. Use single backslashes: `replace(/[\$,]/g, '')`.